Template/LO/SubProcesses/setcuts.f

```diff
@@ -8,6 +8,7 @@ SUBROUTINE SETCUTS
 c
       include 'genps.inc'
       include 'nexternal.inc'
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc